src/styles/index.scss

```diff
@@ -1,3 +1,52 @@
+@import '/src/styles/variables';
+
 body {
+  @include center-all;
+
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc((7 * $field-side) + (6 * $calendar-gap));
```

Comment:
It would be better to move the width property to the `.calendar__field` class. This way, the width of the calendar will adjust automatically based on the number of child elements. You can calculate the width of the `.calendar__field` by multiplying the `$field-side` and `$calendar-gap` variables by 7 (the number of days in a week). Then, in the `.calendar` class, you can just set `width: max-content;` to make it fit the width of its content.